src/proteus/interior/aragog.py

```diff
@@ -64,6 +64,14 @@ def RunAragog(config:Config, dirs:dict, IC_INTERIOR:int, hf_row:dict, hf_all:pd.
     else:
         UpdateAragogSolver(dt, hf_row)
 
+    # Update tidal heating inside Aragog
+    tidal_value= 0.0
+    if config.interior.tidal_heat:
+        if config.orbit.dummy:
+            tidal_value = config.orbit.dummy.H_tide
+    tidal_value /= aragog_solver.parameters.scalings.power_per_mass
+    aragog_solver.parameters.energy.tidal_value = tidal_value
+
```

Comment:
These lines should go into the `SetupAragogSolver` (done once for all from the config).

Reply:
> I have now moved this, although in the future we will also need to make sure it's updated in the UpdateAragogSolver function, since the tidal heat production will evolve over time.

Make sense, thanks!